js/index.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+console.log("hello newbie, index");
+
+// get from local storage or empty array
+const cards = JSON.parse(localStorage.getItem("cards")) ?? [];
+// call function from renderCards.js
+renderCardsConditionally(cards, false);
```

Comment:
this calls your function with a second argument, but your function only accepts one parameter